src/Bridge/Symfony/Bundle/Resources/config/symfony_uid.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="api_platform.identifier.symfony_ulid_normalizer" class="ApiPlatform\Core\Bridge\Symfony\Identifier\Normalizer\UlidNormalizer" public="false">
```

Comment:
Why not make the two Normalizers in one? 
Like this : https://github.com/symfony/symfony/blob/dafcc58e6bd8553a6f461ff585322206ffd81956/src/Symfony/Component/Serializer/Normalizer/UidNormalizer.php#L40-L55